### DISCUSSION TITLE
More disabilty in the model or data

### DISCUSSION THREAD

**Discussion by Beastmaster23** [hlky/stable-diffusion#271] [General]
#### Hello!
I love what you are doing with this AI. I really want to use this for art or even game art. I'm disabled so I can't draw particully well on things bigger then 32x32. Pretty limiting, but this could really open the doors for me pun intended. This leads to me thinking of coxing the AI to draw me in a my wheelchair, but it's clear that it wasn't trained with enough disabled equipment. Because no matter what keywords I use it produces manual wheelchairs. So maybe add more objects to the dataset? Thank you for reading my tedtalk

**Comment by Kyxsune** [comment]
The dataset this model was trained on was put together by Stability AI. As far as I'm aware the maintainers of this repository are not involved in its curation. 

Nonetheless, I think perhaps enabling and using textual inversion may help out. 

https://github.com/hlky/sd-enable-textual-inversion

I haven't done it myself yet.

**Reply by Beastmaster23** [reply]
Seems pretty difficult, but I will give it a try

**Comment by EliEron** [comment]
To elaborate on the existing answer, the data comes from the [LAION-5B](https://laion.ai/blog/laion-5b/) dataset. Which is a dataset containing over 5 Billion images. Though the current Stable-Diffusion model was trained on the English language subset of that data which is composed of 2.3 Billion images.

The more appropriate place to bring up this subject would be in the official [Stable-Diffusion](https://github.com/CompVis/stable-diffusion) repository, though I wouldn't hold my breath about getting a response. There are a lot of open issues on that repository and I suspect the developers are spending most of their time actually working on the model. And to be honest I have a feeling it won't really be all that simple to fix this issue in the first place. When dealing with datasets this massive it goes without saying that just adding a couple extra photos of an object won't exactly change much.

Your best best is to just wait for future model versions and hope that the model ends up getting better at generating disability content. They are currently working on an update to the model which should hopefully be released in the not too distance future.
